### ISSUE TITLE
To distinguish single and multiple modifiers

### ISSUE THREAD

**Issue by spring-haru** [kana/vim-arpeggio#17]
`vim` and `gvim` (and `neovim`?) do not seem to be able to distinguish between `<C-x>` and `<C-S-x>` (multiple modifiers). For example, see [here](https://stackoverflow.com/questions/1506764/how-to-map-ctrla-and-ctrlshifta-differently). I wonder if `arpeggio` can solve the problem?

**Comment by kana** [comment]
No. vim-arpeggio is just a Vim plugin. What Vim can recognize is what vim-arpeggio can recognize.
